packages/back-end/src/routers/saved-group/saved-group.controller.ts

```diff
@@ -96,11 +57,29 @@ export const postSavedGroup = async (
     type,
     condition,
     description,
+    projects,
   } = req.body;
 
-  if (!context.permissions.canCreateSavedGroup()) {
+  if (!context.permissions.canCreateSavedGroup({ ...req.body })) {
     context.permissions.throwPermissionError();
   }
+
+  if (projects) {
+    await Promise.all(
+      projects.map(async (projectId) => {
+        // Ensure the project exists
+        const project = await context.models.projects.getById(projectId);
```

Comment:
Hmm should our `getById` contextual methods filter by request org by default? Or they do already do that and then lines 76-78 are unnecessary? From what I can tell quickly looking at the code it looks like they don't

Reply:
I remember seeing "always filter by org" as one of the important query standards (for security), so if we could have that built in that seems ideal. Maybe Jeremy can weigh in on the best way to make that happen?

Reply:
cc: @jdorn

Reply:
All of the `context.models` automatically filter by org id, so just doing `getById(projectId)` is fine.   It will actually do a query for `{organization: context.org.id, id: projectId}` under the hood